File: encuestas-frontend/src/app/shared/models/tipo-pregunta.ts
// Tipo de cada pregunta dentro de una encuesta (backend: TipoPregunta.java)
export enum TipoPregunta {
  TEXTO_LIBRE = 'TEXTO_LIBRE', // respuesta abierta
  OPCION_UNICA = 'OPCION_UNICA', // elige 1 opción (radio button)
  OPCION_MULTIPLE = 'OPCION_MULTIPLE', // elige varias opciones (checkbox)
  ESCALA = 'ESCALA',
  EMAIL = 'EMAIL',
  NUMERO = 'NUMERO',
  TELEFONO = 'TELEFONO', // ejemplo: del 1 al 5
}

File: encuestas-backend/src/main/java/com/encuestas/encuestas_backend/model/TipoPregunta.java
package com.encuestas.encuestas_backend.model;

public enum TipoPregunta {
    TEXTO_LIBRE,      // el encuestado escribe una respuesta abierta
    OPCION_UNICA,     // elige 1 de varias opciones (radio button)
    OPCION_MULTIPLE,  // elige varias opciones (checkbox)
    ESCALA,         // por ejemplo, del 1 al 5
    EMAIL,
    NUMERO,
    TELEFONO
}
